app/uk/gov/hmrc/hec/services/FileStoreService.scala

```diff
@@ -63,7 +64,7 @@ class FileStoreServiceImpl @Inject() (client: PlayObjectStoreClient, config: Con
       client
         .putObject(
           path = Path.Directory(dirName).file(fileName),
-          content = fileContent,
+          content = Source.fromBytes(fileContent.getBytes("US-ASCII"), "UTF-8").mkString,
```

Comment:
is it not possible to call `getBytes` with `UTF-8` here?

Reply:
This has been proved now